src/tests/ftest/pool/pda.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+
+from apricot import TestWithServers
+
+class PoolPDAProperty(TestWithServers):
+    # pylint: disable=too-many-ancestors
+    # pylint: disable=too-few-public-methods
+    """run tests with pool pda property
+
+    Test Class Description: To validate pool pda works properly
+
+    :avocado: recursive
+    """
+    def verify_cont_ec_pda(self, expected_value):
+        """
+        Verify the container ec_pda property.
+
+        Args:
+            expected_value (int): expected container ec_pda value
+        """
+        daos_cmd = self.get_daos_command()
+        cont_prop = daos_cmd.container_get_prop(self.pool.uuid,
+                                                self.container.uuid)
+        cont_prop_stdout = cont_prop.stdout_text
+        prop_list = cont_prop_stdout.split('\n')[1:]
+        cont_index = [i for i, word in enumerate(prop_list)
+                      if word.startswith('Performance domain affinity level of EC')][0]
+        pda_str = (prop_list[cont_index].split('Performance domain affinity level of EC')[1].strip())
+        pda_value = int(pda_str)
+        self.assertEqual(expected_value, pda_value)
+
+    def verify_cont_rp_pda(self, expected_value):
+        """
+        Verify the container rp_pda property.
+
+        Args:
+            expected_value (int): expected container rp_pda value
+        """
+        daos_cmd = self.get_daos_command()
+        cont_prop = daos_cmd.container_get_prop(self.pool.uuid,
+                                                self.container.uuid)
+        cont_prop_stdout = cont_prop.stdout_text
+        prop_list = cont_prop_stdout.split('\n')[1:]
+        cont_index = [i for i, word in enumerate(prop_list)
+                      if word.startswith('Performance domain affinity level of RP')][0]
+        pda_str = (prop_list[cont_index].split('Performance domain affinity level of RP')[1].strip())
+        pda_value = int(pda_str)
+        self.assertEqual(expected_value, pda_value)
+
+    def test_pda_pool_property(self):
+        """Jira ID: DAOS-9550.
+
+        Test Description:
+            Verify container pda is picked from container property
+            and not the pool default pda.
+
+        Use Case:
+            Create the Pool with default.
+	    Create the pool with specified ec_pda,rp_pda
+            Create the container and verify the pda is same for both
+                pool and container.
+            Create the container with different pda value
+            Verify the pda is updated for the container.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=small
+        :avocado: tags=pool,pool_pda_property
```

Comment:
`small` isn't necessary, since it's really only used with `hw,small`, but that's okay.
This is the format we've been trying to follow:
```suggestion
        :avocado: tags=all,full_regression
        :avocado: tags=vm
        :avocado: tags=pool
        :avocado: tags=pool_pda_property
```

Reply:
Sure, let me try this to fix this